### PR TITLE
Docs for Linear Pool Subgraph Integrations

### PR DESCRIPTION
**Pull request by BankzAndBittle**
Adds documentation for updating the subgraph with new linear pool deployments

### CODE REVIEW

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 1]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
 
 If all goes well, you should only need to change the fields highlighted above.
 
+## Subgraph
+Adding new integrations to the subgraph is important for tracking linear pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
+
+In the `abi` folder add two JSON files. One containing the new linear pool factory's abi and the next containing the new linear pool's abi.
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
In the `abi` folder, add two JSON files: one for the new Linear Pool ABI and one for the corresponding factory ABI.
```

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 2]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
 
 If all goes well, you should only need to change the fields highlighted above.
 
+## Subgraph
+Adding new integrations to the subgraph is important for tracking linear pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2).
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
Adding new integrations to the subgraph is important for tracking Linear Pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
```

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 3]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
 
 If all goes well, you should only need to change the fields highlighted above.
 
+## Subgraph
+Adding new integrations to the subgraph is important for tracking linear pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
+
+In the `abi` folder add two JSON files. One containing the new linear pool factory's abi and the next containing the new linear pool's abi. 
+
+Open the `networks.yaml` file and add an entry for each network that the new linear pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`.
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
In `networks.yaml`, add an entry for each network that the new Linear Pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`. 
```

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 4]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
...
 If all goes well, you should only need to change the fields highlighted above.
 
+## Subgraph
+Adding new integrations to the subgraph is important for tracking linear pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
+
+In the `abi` folder add two JSON files. One containing the new linear pool factory's abi and the next containing the new linear pool's abi. 
+
+Open the `networks.yaml` file and add an entry for each network that the new linear pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`. 
+
+Add the linear pool to the `PoolType` namespaces within `src/mappings/helpers/pool.ts`. In the `pool.ts` file, update `hasVirtualSupply` and `isLinearPool` to include the new `PoolType`.
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
Add the Linear Pool to the `PoolType` namespaces within `src/mappings/helpers/pool.ts`. In `pool.ts`, update `hasVirtualSupply` and `isLinearPool` to include the new `PoolType`. 
```

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 5]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
...
+## Subgraph
+Adding new integrations to the subgraph is important for tracking linear pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
+
+In the `abi` folder add two JSON files. One containing the new linear pool factory's abi and the next containing the new linear pool's abi. 
+
+Open the `networks.yaml` file and add an entry for each network that the new linear pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`. 
+
+Add the linear pool to the `PoolType` namespaces within `src/mappings/helpers/pool.ts`. In the `pool.ts` file, update `hasVirtualSupply` and `isLinearPool` to include the new `PoolType`. 
+
+In `manifest.template.yaml`, copy and paste the `ERC4626LinearPoolV4Factory` entry and change to fit the new linear pool's protocol and version. Verify the events match the source code.
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
In `manifest.template.yaml`, copy and paste the `ERC4626LinearPoolV4Factory` entry and change to fit the new Linear Pool's protocol and version. Verify the events match the source code. 
```

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 6]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
...
+
+In the `abi` folder add two JSON files. One containing the new linear pool factory's abi and the next containing the new linear pool's abi. 
+
+Open the `networks.yaml` file and add an entry for each network that the new linear pool is deployed on. This includes information on the `network`, factory name, `address` of the deployed contract, and `startBlock`. Maintain the current naming conventions by naming the factory, `<protocol>LinearPool<version>Factory`. 
+
+Add the linear pool to the `PoolType` namespaces within `src/mappings/helpers/pool.ts`. In the `pool.ts` file, update `hasVirtualSupply` and `isLinearPool` to include the new `PoolType`. 
+
+In `manifest.template.yaml`, copy and paste the `ERC4626LinearPoolV4Factory` entry and change to fit the new linear pool's protocol and version. Verify the events match the source code. 
+
+Add event handler mappings for the the factory in `poolFactory.ts`. Copy and paste `handleNewERC4626LinearPoolV4` and make necessary changes for the new linear pool's protocol and version. The function name must match the name provided in the factory's handler from `manifest.template.yaml`.
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
Add event handler mappings for the factory in `poolFactory.ts`. Copy and paste `handleNewERC4626LinearPoolV4` and make necessary changes for the new Linear Pool's protocol and version. The function name must match the name provided in the factory's handler from `manifest.template.yaml`.
```

**Review by gerrrg — CHANGES_REQUESTED** [review]
Minor language edits and capitalizations

File: docs/guides/boosted-pool-creators/new-protocol.md [hunk 7]
```diff
@@ -83,5 +83,18 @@ Adapt the test file to the protocol name, and change the block number of the tes
 
 If all goes well, you should only need to change the fields highlighted above.
 
+## Subgraph
+Adding new integrations to the subgraph is important for tracking Linear Pool creations. The subgraph is maintained in the [Balancer Subgraph Repository](https://github.com/balancer/balancer-subgraph-v2). 
+
+In the `abi` folder add two JSON files: one for the new Linear Pool ABI and one for the the corresponding factory ABI.
```
Comment (gerrrg) [inline docs/guides/boosted-pool-creators/new-protocol.md]:
```suggestion
In the `abi` folder, add two JSON files: one for the new Linear Pool ABI and one for the the corresponding factory ABI. 
```

**Review by gerrrg — APPROVED** [review]
Looks good now. I added a suggestion to add one comma, but approving now to unblock you

**Review by gerrrg — APPROVED** [review]

**Review by mikemcdonald — APPROVED** [review]

**Pull request merged**